modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/configuration/admin/service/OpenIdConnectProviderManagedServiceFactory.java

```diff
@@ -459,7 +464,7 @@ defaultUserId, _generateMetadataJSON(properties),
 
 	private void _updateOAuthClientEntry(
 		long companyId, String oldProviderName,
-		Dictionary<String, ?> properties) {
+		Dictionary<String, ?> oldProperties) {
```

Comment:
`oldProperties` is a bit misleading as in an earlier function call not old properties are passed. In my opinion it would be better to pass both `properties` and `oldProperties` (or null) and check which one to use for `_oAuthClientEntryLocalService.fetchOAuthClientEntry` call.